config/crd/bases/nfd.kubernetes.io_nodefeaturediscoveries.yaml

```diff
@@ -0,0 +1,111 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+    api-approved.kubernetes.io: "unapproved, experimental-only"
+  creationTimestamp: null
+  name: nodefeaturediscoveries.nfd.kubernetes.io
+spec:
+  group: nfd.kubernetes.io
+  names:
+    kind: NodeFeatureDiscovery
+    listKind: NodeFeatureDiscoveryList
+    plural: nodefeaturediscoveries
+    singular: nodefeaturediscovery
+  scope: Namespaced
```

Comment:
just a thought but I was wondering should the CRD be cluster scoped instead of `Namespaced`.

Reply:
this is the same CRD , this patch intention is about the operator framework layout, changes to the CRD will come in a following PR